gpu_ocean/SWESimulators/FBL.py

```diff
@@ -195,8 +195,11 @@ def fromfilename(cls, gpu_ctx, filename, cont_write_netcdf=True):
         timeIntegrator = sim_reader.get("time_integrator")
         y_zero_reference_cell = sim_reader.get("y_zero_reference_cell")
 
-        wind_stress_type = sim_reader.get("wind_stress_type")
-        wind = Common.WindStressParams(type=wind_stress_type)
+        try:
```

Comment:
Again, I think we should require that the writer also writes out the stress type

Reply:
See comment above